example_with_targets/src/lib.rs

```diff
@@ -1,31 +0,0 @@
-use shopify_function::prelude::*;
-use shopify_function::Result;
-
-#[shopify_function_target(
-    // Implicit target = "example.target-a"
-    // Implicit generated module name = "target_a"
-    query_path = "a.graphql",
-    schema_path = "schema.graphql"
-)]
-fn target_a(
-    _input: target_a::input::ResponseData,
-) -> Result<target_a::output::FunctionTargetAResult> {
-    Ok(target_a::output::FunctionTargetAResult { status: Some(200) })
-}
-
-#[shopify_function_target(
-    // Explicit target if function name does not match target handle
-    target = "example.target-b",
```

Comment:
With the new `shopify_function` macro, we're missing this functionality. Arguably, it's not very important though because you can configure the mapping between target and export via your TOML file. I don't have a strong conviction for keeping this behaviour, but it might worth getting another set of eyes on it.